postman.json

```diff
@@ -1119,18 +1242,18 @@
 						{
 							"key": "older_than",
 							"value": "Integer",
-							"description": "Expects an order ID and returns orders with and ID < the passed ID",
+							"description": "Expects an order ID and returns orders with an ID < the passed ID",
 							"disabled": true
 						},
 						{
 							"key": "newer_than",
 							"value": "Integer",
-							"description": "Expects an order ID and returns orders with and ID > the passed ID",
+							"description": "Expects an order ID and returns orders with an ID > the passed ID",
 							"disabled": true
 						}
 					]
 				},
-				"description": "Fetches the last 200 orders made by your account with all the trades associated.\n\nIf you need more than the last 200 trades use the `older_than` param. Passing an order ID will return results only orders which are older than that order ID."
+				"description": "Fetches the last 200 orders made by your account with all the trades associated.\n\nIf you need more than the last 200 orders use the `older_than` param. Passing an order ID will return results only orders which are older than that order ID.\n\n<blockquote><br>\n<p>Examples </p>\n</blockquote>\n\n``` python\n\"\"\"\nOrder IDs are strictly increasing, so code like the following\nshort example may be used to detect new orders.\n\"\"\"\n\nlatest_order_id = 0\nwhile True:\n\tresult = api.get(\"/v1/user/orders\", params={\"newer_than\": latest_order_id}).json()\n\tif result['data']['orders'] != []:\n\t\t## process the new order ##\n\t\tlatest_order_id = result['data']['orders'][0]['id']\n\n\n\"\"\"\nIf you're searching for an order older than the latest\n200, you can use older_than to iterate over old orders.\n\"\"\"\n\norders = api.get(\"/v1/user/orders\").json()\nsearch_price = \"4.99896025\"\nwhile True:\n\tfor order in orders[\"data\"][\"orders\"]:\n\t\tif order[\"price\"] == search_price:\n\t\t\treturn order\n\toldest_order_id = orders[\"data\"][\"orders\"][-1][\"id\"]\n\torders = api.get(\"/v1/orders\", params={\"older_than\": oldest_order_id}).json()\n\n```"
```

Comment:
Since this example is python specific (and for now there are no other language examples) for now we should show the 'examples' header only for python using the 'lang-specific' class provided by slate